src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Clip/ButtonClippingTestsControl.xaml.cs

```diff
@@ -1,28 +0,0 @@
-using System;
```

Comment:
That's something done by git. I was a rename on my side, but I don't know why it's presenting it as a remove + add.

Reply:
Probably because you changed a lot of the code in the file, git gets confused in those cases. It's generally better to commit a file rename only, then another commit with edits. (Not a big deal in this case because the history is not that interesting for the samples)